v2/flagship-events/src/main/java/com/keap/dataflow/flagshipevents/FlagshipEventsPubsubToBigQuery.java

```diff
@@ -171,7 +174,8 @@ public PCollection<Iterable<String>> expand(PCollection<String> input) {
               new SimpleFunction<>() {
                 @Override
                 public Iterable<String> apply(String input) {
-                  String[] splitInput = input.split("&");
+                  String[] splitInput =
+                      input.split("(?<=})&"); // only ampersands following a closing brace
```

Comment:
This was the thing causing the issues, since a role for some reason had an ampersand in it and caused the flow to throw an unparsable error.